### PR TITLE
Adding myPOS2017 dataset

### PR DESCRIPTION
**Pull request by hungluumfc**
myPOS Corpus (Myanmar Part-of-Speech Corpus) for Myanmar language NLP Research and Developments

### CODE REVIEW

**Review by lhoestq — COMMENTED** [review]
Can you regenerate the dataset_infos.json file please ?

It looks like the checksum of the `test.txt` file is not the same as the one that is currently available on the myPOS github repo.

```
datasets-cli test ./datasets/mypos --save_infos --ignore_verifications --all_configs
```

File: datasets/mypos/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+### Languages
+
+[Needs More Information]
+
+## Dataset Structure
+
+### Data Instances
+
+[Needs More Information]
```
Comment (lhoestq) [inline datasets/mypos/README.md:71]: Can you add an example from the dataset here please ?

You can get one using `load_dataset` and taking the first sample `dataset["train"][0]` for example

File: datasets/mypos/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+myPOS Corpus (Myanmar Part-of-Speech Corpus) for Myanmar language NLP Research and Developments
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+[Needs More Information]
```
Comment (lhoestq) [inline datasets/mypos/README.md:65]: can you add the language here please ?

File: datasets/mypos/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Needs More Information]
+- **Repository:** [Needs More Information]
+- **Paper:** [Needs More Information]
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Needs More Information]
```
Comment (lhoestq) [inline datasets/mypos/README.md:53]: Can you fill those fields if applicable ?

File: datasets/mypos/README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+[Needs More Information]
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+[Needs More Information]
```
Comment (lhoestq) [inline datasets/mypos/README.md:139]: can you add the citation here please ?

File: datasets/mypos/mypos.py [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    def __init__(self, **kwargs):
+        """BuilderConfig for myPOS2017.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(myPOS2017Config, self).__init__(**kwargs)
+
+
+class myPOS2017(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/mypos/mypos.py:55]:
```suggestion
class Mypos(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `mypos`

you'll have to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one ! :) 

I left a few comments

**Comment by hungluumfc** [comment]
I've updated the code and Readme to reflect your comments.
Thank you very much,

**Comment by lhoestq** [comment]
looks like this PR includes changes about many other files than the ones for myPOS2017

Could you open another branch and another PR please ?
(or fix this branch)

File: datasets/mypos/README.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+### Data Instances
+
+{'id': '0', 'pos_tags': [6, 6, 9, 6, 3, 6, 8, 9, 14, 8, 9, 8, 14, 8, 9, 11], 'tokens': ['ယခု', 'လ', 'တွင်', 'ပျားရည်', 'နှင့်', 'ပျားဖယောင်း', 'များ', 'ကို', 'စုဆောင်း', 'ကြ', 'သည်', 'ဟု', 'ခန့်မှန်း', 'နိုင်', 'သည်', '။']}
+
+### Data Fields
+
+- `id`: a `string` feature.
+- `tokens`: a `list` of `string` features.
+- `pos_tags`: a `list` of classification labels, with possible values including `abb`, `adj`, `adv`, `conj`, `fw`.
```
Comment (lhoestq) [inline datasets/mypos/README.md:84]: could you add the full list of tags here please ?

**Comment by lhoestq** [comment]
Hi @hungluumfc ! Have you had a chance to fix this PR so that it only includes the changes for `mypos` ? 

Feel free to ping me if you have questions or if I can help :)

**Pull request closed without merging**